src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -5128,27 +5128,47 @@ private void OnDecorationBoundingBoxChanged(int x, int y, int width, int height)
         }
         private void OnInputColorChanged(float x, float y, float z, float w)
         {
-            InputColor = new Vector4(x, y, z, w);
+            ReusablePool<Vector4>.GetOne((vector4, x, y, z, w) =>
+            {
+                vector4.Reset(x, y, z, w);
+                InputColor = vector4;
+            }, x, y, z, w);
```

Comment:
This code is incorrect.
The vector4 passed as a delegate argument in ResuablePool is an object that should only be used temporarily.
However, in this code, a reference to this vector4 is stored in InputColor, causing it to be used for a longer duration instead of being used temporarily.

in this code, you can't use pool

Reply:
If corrected properly
```suggestion
            ReusablePool<Vector4>.GetOne((vector4, x, y, z, w) =>
            {
                vector4.Reset(x, y, z, w);
                SetInternalInputColor(vector4);
            }, x, y, z, w);
```

Reply:
Original code:
InputColor = vector4;

will call InputColor Setter:
```
public Vector4 InputColor
{
    ...
    set
    {
        if (NUIApplication.IsUsingXaml)
        {
            SetValue(InputColorProperty, value);
        }
        else
        {
            SetInternalInputColor(value);
        }
        NotifyPropertyChanged();
    }
}
```

If SetInternalInputColor(vector4) is called directly, NotifyPropertyChanged is not called.

Reply:
if you need to call `NotifyPropertyChanged` call it after `SetInternalInputColor` 


``` c#
            ReusablePool<Vector4>.GetOne((vector4, x, y, z, w) =>
            {
                vector4.Reset(x, y, z, w);
                SetInternalInputColor(vector4);
                NotifyPropertyChanged("InputColor");
            }, x, y, z, w);
```

Reply:
The review I intended to give is that you are not using the Vector4 temporarily, it should be used only on a temporary

Reply:
Done.